server.js

```diff
@@ -18,6 +19,31 @@ app.get('/', (req, res) => {
     web.proxy(req, res)
 })
 
+app.post('/login', (req, res, next) => {
+  //this user need be dynamic
+  const user ={
+    id: "1",
+    username: 'username'
+  }
+    var token = jwt.sign({user: user}, 'secretarg', {
+      expiresIn: 300 // expires in 5min
+    });
+    res.status(200).send({ auth: true, token: token });
+
+  res.status(500).send('Invalid login!');
+})
+
+function verifyJWT(req, res, next){
+  //verify if has token on header
+  var token = req.headers['x-access-token'];
+  if (!token) return res.status(401).send({ auth: false, message: 'No token provided.' });
+
+  jwt.verify(token, 'secretarg', function(err, decoded) {
```

Comment:
```suggestion
  jwt.verify(token, SECRET_KEY, function(err, decoded) {
```

Reply:
ty for suggest, i make a new commit with this suggests and anothers modifications